### PR TITLE
i18n(es): update `rendering-modes` & `community-resources/content`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

updates `rendering-modes` & `community-resources/content`

#### Related issues & labels (optional)

- Closes # N/A
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/es/basics/rendering-modes.mdx [hunk 1]
```diff
@@ -33,12 +34,14 @@ El modo `static` predeterminado de Astro es una opción potente y moderna para s
...
+Con [un adaptador SSR](/es/guides/server-sider-rendering/), las otras dos opciones de salida de Astro se pueden configurar para habilitar el **renderizado bajo demanda de algunas o todas tus páginas, rutas o endpoints de API**:
   - __`output: 'server'`__ para sitios altamente dinámicos con la mayoría o todas las rutas bajo demanda.
   - __`output: 'hybrid'`__ para sitios principalmente estáticos con algunas rutas bajo demanda.
   
 Desde que son generadas por visita, estas rutas pueden ser personalizadas para cada visitante. Por ejemplo, una página renderizada bajo demanda puede mostrar a un usuario registrado su información de cuenta o mostrar datos actualizados recientemente sin requerir una reconstrucción completa del sitio. El renderizado bajo demanda en el servidor en el momento de la solicitud también se conoce como **renderizado del lado del servidor (SSR)**.
 
+<ReadMore>Lee más sobre las [características disponibles para ti con el renderizado bajo demanda y un adaptador](/es/guides/server-side-rendering/#características-de-renderizado-bajo-demanda), como cookies, objetos Response y Request, streaming de HTML y más.</ReadMore>
```
Comment (dreyfus92) [inline src/content/docs/es/basics/rendering-modes.mdx:43]:
```suggestion
<ReadMore>Lee más sobre las [características disponibles para ti con el renderizado bajo demanda y un adaptador](/es/guides/server-side-rendering/#características-de-renderización-bajo-demanda), como cookies, objetos Response y Request, streaming de HTML y más.</ReadMore>
```

File: src/content/docs/es/basics/rendering-modes.mdx [hunk 2]
```diff
@@ -33,12 +34,14 @@ El modo `static` predeterminado de Astro es una opción potente y moderna para s
 
 ### Renderizado bajo demanda
 
-Las otras dos opciones de salida de Astro se pueden configurar para habilitar el **renderizado bajo demanda de algunas o todas tus páginas, rutas o endpoints de API**:
+Con [un adaptador SSR](/es/guides/server-sider-rendering/), las otras dos opciones de salida de Astro se pueden configurar para habilitar el **renderizado bajo demanda de algunas o todas tus páginas, rutas o endpoints de API**:
```
Comment (dreyfus92) [inline src/content/docs/es/basics/rendering-modes.mdx:37]:
```suggestion
Con [un adaptador SSR](/es/guides/server-side-rendering/), las otras dos opciones de salida de Astro se pueden configurar para habilitar el **renderizado bajo demanda de algunas o todas tus páginas, rutas o endpoints de API**:
```

**Comment by yanthomasdev** [comment]
Heads up we're missing this commit here: https://github.com/withastro/docs/commit/094896e2e83e90dbbfeda747976cf2975ec34cdd

**Comment by dreyfus92** [comment]
Thanks for the review @yanthomasdev 😁

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**